pkgs/development/tools/supabase-cli/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, buildGo118Module
```

Comment:
```suggestion
, buildGoModule
```

Why do we need Go 1.18? If this is really required this must be commented.

Reply:
works with 1.19, must have misread the docs of supabase